addons/website_slides/view/website_slides_embed.xml

```diff
@@ -187,22 +187,22 @@
                                 <div class="col-lg-8 col-md-8 col-sm-8 col-xs-8">
                                     <div class="col-lg-4 col-md-4 col-sm-4 col-xs-4 text-center"></div>
                                     <div class="col-lg-1 col-md-1 col-sm-1 col-xs-1 text-center">
-                                        <span id="first" title="First slide">
+                                        <span id="first" class="hide slide_nav_button" title="First slide">
```

Comment:
@jem-odoo If we do this, the ```nav_buttons``` are first seen and then hide that's why I think we have to use the ```.hide``` class in div and then we have can show the ```div``` using ```show()``` method. Is it better for you ???